uai2017/python/create-bidding-invitation.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+
+###############################################################################
+# Not to spec
+###############################################################################
+
+## Import statements
+import argparse
+import csv
+import getpass
+import sys
+import re
+import openreview
+import requests
+from uaidata import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('type', help="the type of the bidding Reviewer or AreaChair")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+else:
+    client = openreview.Client(baseurl=args.baseurl)
+baseurl = client.baseurl
+
+COCHAIRS = UAIData.get_program_co_chairs()
+PC = UAIData.get_program_committee()
+SPC = UAIData.get_senior_program_committee()
```

Comment:
the PR I submitted a few minutes ago changes this - these three lines should be deleted (32, 33, and 34)